docs/extend/index.md

```diff
@@ -97,7 +97,7 @@ started without error.
     ```bash
     $ docker volume ls
 
-    DRIVER              NAME
+    DRIVER              VOLUME NAME
     vieux/sshfs         sshvolume
```

Comment:
from test, command "docker volume ls" output “VOLUME NAME”, so it should be fixed all "NAME" to "VOLUME NAME", is that right?